### PR TITLE
Create pygame.yml

### PR DESCRIPTION
**Pull request by illume**
hey hey. Another project for your fine website.

### CODE REVIEW

**Comment by shiftkey** [comment]
```
1 files found with errors:
 - _data/projects/pygame.yml:
    - Rename tag 'games' to be 'game'
```

File: _data/projects/pygame.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+name: pygame
+desc: games, art, music and even sciencing with python.
+site: https://pygame.org/
+tags:
+  - pygame
+  - python
+  - games
```
Comment (ritwik12) [inline _data/projects/pygame.yml]:
```suggestion
  - game
```

**Review by ritwik12 — CHANGES_REQUESTED** [review]

File: _data/projects/pygame.yml (hunk 1 above)
Comment (illume) [reply _data/projects/pygame.yml]: Thanks :)

**Review by ritwik12 — APPROVED** [review]

**Comment by ritwik12** [comment]
@illume Thanks !!

**Pull request merged**